packages/firebase_crashlytics/README.md

```diff
@@ -91,20 +122,21 @@ void main() {
 }
 ```
 
-Overriding `FlutterError.onError` with `Crashlytics.instance.recordFlutterError`  will automatically catch all 
-errors that are thrown from within the Flutter framework.  
-If you want to catch errors that occur in `runZoned`, 
-you can supply `Crashlytics.instance.recordError` to the `onError` parameter:
+Overriding `FlutterError.onError` with `Crashlytics.instance.recordFlutterError`  will automatically catch all errors that are thrown from within the Flutter framework.
+
+If you want to catch errors that occur in `runZoned`, you can supply `Crashlytics.instance.recordError` to the `onError` parameter:
+
 ```dart
 runZoned<Future<void>>(() async {
```

Comment:
Latest stable release for Flutter is now 1.17 and `runZoned` is deprecated and that [`runZonedGuarded`](https://api.dart.dev/stable/2.8.1/dart-async/runZonedGuarded.html) should be used instead